libflux/flux-core/Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "flux-core"
 version = "0.154.0"
+rust-version = "1.68"
```

Comment:
Errrrrrrrr this says 1.68 but the PR title says 1.78?

Reply:
The version that we build with (from `rust-toolchain.toml`) is 1.68. I didn't really want to change that. As I understand it this sets the minimum supported version for the crate. Though we build with 1.68 it seems others are building influxdb with newer versions (#5479) so we want to support those too.